src/app/saga.ts

```diff
@@ -279,8 +281,21 @@ export function* handleDeepLink(action: OpenDeepLink) {
 
   const rawParams = parse(deepLink)
   if (rawParams.path) {
-    const pathParts = rawParams.path.split('/')
-    if (rawParams.path.startsWith('/v/')) {
+    // android resolves dynamic links to the `link`, iOS handles the plain
+    // dynamic link
+    if (
+      rawParams.hostname === new URL(DYNAMIC_LINK_DOMAIN_URI_PREFIX).hostname ||
+      rawParams.hostname === new URL(WEB_LINK).hostname
```

Comment:
If it's the WEB_LINK (https://valoraapp.com), no need to resolve it. Or am I missing something?

Reply:
no, you're right. i had left it there because it didn't hurt, but i've updated now to not resolve if it's the resolved link already